### PR TITLE
Fix undefined state on first render after dynamicially loading module in react-redux v6

### PR DESCRIPTION
**Pull request by navneet-g**
See #27 
The primary change is in DynamicModuleLoader.tsx. The rest is adding another sample project that repro'd the issue.

https://github.com/Microsoft/redux-dynamic-modules/pull/59/files#diff-3ceb15cf807c230eeddb28876224070f

### CODE REVIEW

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 1]
```diff
@@ -98,21 +103,46 @@ class DynamicModuleLoaderImpl extends React.Component<
                     "Store could not be resolved from React context"
                 );
             }
+        } else {
+            // We will add modules dynamically and due to github issue https://github.com/Microsoft/redux-dynamic-modules/issues/27#issuecomment-464905893
+            // The very first render will not get latest state, to fix that we will need to get latest state from store directly on first render
+            this._getLatestState = ReactReduxContext;
```
Comment (abettadapur) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: This weird, can we rename this variable somehow?

`useReactRedux6Context`

File: packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx [hunk 2]
```diff
@@ -98,21 +103,46 @@ class DynamicModuleLoaderImpl extends React.Component<
...
+        } else {
+            // We will add modules dynamically and due to github issue https://github.com/Microsoft/redux-dynamic-modules/issues/27#issuecomment-464905893
+            // The very first render will not get latest state, to fix that we will need to get latest state from store directly on first render
+            this._getLatestState = ReactReduxContext;
         }
 
         this._addedModules = store.addModules(modules);
     }
 
+    private _wrappedRender = () => {
```
Comment (abettadapur) [inline packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: Again, something to denote that this is the RR6 path?

Comment (navneet-g) [reply packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx]: Sure will rename

**Comment by navneet-g** [comment]
> Why are you incrementing version now?

this is new  published version as I published to npm

**Comment by abettadapur** [comment]
Why did you publish before closing this PR??

**Comment by navneet-g** [comment]
> Why did you publish before closing this PR??

You had already approved, and the publish happened in the same branch not in master

**Comment by abettadapur** [comment]
for the future, we should wait till we merge to master and always publish from them. IMO, its bad practice to be publishing to npm from feature branches

**Comment by navneet-g** [comment]
> for the future, we should wait till we merge to master and always publish from them. IMO, its bad practice to be publishing to npm from feature branches

Sure.

**Comment by abettadapur** [comment]
I modified the lerna.json file to prevent this in the future :)

**Pull request merged**